ZAPD/ZAPD.vcxproj

```diff
@@ -119,6 +119,7 @@ mkdir build\ZAPD
       <Profile>true</Profile>
       <AdditionalDependencies>ZAPDUtils.lib;/WHOLEARCHIVE:ExporterExample.lib;%(AdditionalDependencies)</AdditionalDependencies>
       <IgnoreAllDefaultLibraries>false</IgnoreAllDefaultLibraries>
+      <SubSystem>Console</SubSystem>

```

Comment:
Was this intentional?

Reply:
Yes. It makes it so the console window doesn't immediately close when the program exists.